.changeset/brown-roses-drive.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/sequence": minor
+---
+
+Incorporate change and change properties into the same method, and update the change signature to match the new add signature.
+
+Instead of having 2 separate methods to change the endpoints of an interval and a parameter, combine them into a single method that will change the endpoints, properties, or both, depending on the arguments passed in. The signature of this combined method is now updated as well, so the new usage of the change method is to call it with an interval id as the first parameter and an object containing the desired portions of the interval to update as the second parameter. For the object parameter, the endpoints field should be an object containing the new start and end values for the interval, and the properties field should be an object containing the new properties for the interval. Either the endpoints field or the properties field can be omitted, but at least one of them must be present. If neither are present, change will return undefined. The new usage of the change method is as follows:
```

Comment:
If we care about enforcing that one of endpoints or properties must be specified, you can use a union type.

Reply:
I don't think it needs to be and nothing bad seems to happen. The description here says "at least one of them must be present" though.